### PR TITLE
added RTLMP* variables and cleaned up per yamllint

### PR DESCRIPTION
**Pull request by jeffng-or**
1. Added docs for some RTLMP* variables
2. Cleaned up content per yamllint
3. Fixed variables in macro_place_util.tcl

### CODE REVIEW

File: flow/scripts/variables.yaml [hunk 1]
```diff
@@ -659,10 +685,136 @@ GLOBAL_ROUTE_ARGS:
...
+  description: >
+    Maximum depth of the physical hierarchy tree.
+  default: 2
+  stages:
+    - floorplan
+RTLMP_MAX_INST:
+  description: >
+    Maximum number of standard cells in a cluster. If unset, rtl_macro_placer
+    will calculate a value based on the design attributes.
+  default: 0
```
Comment (maliberty) [inline flow/scripts/variables.yaml]: Do we need to give a default since unset is handled by OR and 0 isn't informative?  Likewise min/max inst/macro.

Comment (jeffng-or) [reply flow/scripts/variables.yaml]: done!

File: flow/scripts/variables.yaml [hunk 2]
```diff
@@ -108,7 +114,7 @@ PROCESS:
     Technology node or process in use.
 CORNER:
   description: >
-    Library to select based on corner BC/TC/WC.
+    PVT corner library selection. Only available for ASAP7, GF55 and GF180 PDKs.
```
Comment (maliberty) [inline flow/scripts/variables.yaml]: gf55 isn't a public platform so please remove it.  That should fix the scan error

Comment (jeffng-or) [reply flow/scripts/variables.yaml]: removed and pushed up a squashed commit

**Review by maliberty — APPROVED** [review]

**Pull request merged**